### DISCUSSION TITLE
How to implement excel charts, the template with charts does not replace the data

### DISCUSSION THREAD

**Discussion by littlefund** [jxlsteam/jxls#334] [Q&A]
<img width="1083" alt="image" src="https://github.com/user-attachments/assets/db22c38d-d685-48fb-a882-90c53896566e">
what's wrong with my tmplate? does't support excel charts???

**Comment by littlefund** [comment]
If i delete the charts,it works well..

**Reply by SoltauFintel** [reply]
Does that answer your question?

You can see if there is an Excel option "Update data when file is opened"
